README.rst

```diff
@@ -303,6 +303,14 @@ This is the list of operators that can be used:
 - ``ilike``
 - ``in``
 - ``not_in``
+- ``any``
+- ``not_all``
+
+any / not_all
+^^^^^^^^^^^^^
+
+PostgreSQL specific operators can be used to filter queries based on presence
```

Comment:
We could rephrase some parts of this description:
* "based on the presence"
* "of a value in a model attribute of type ``ARRAY``."

Reply:
Implementation has been changed and documentation rephrased.